ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": 
+  {
+    "source": "git",
+    "target": "git://github.com/jenil/bulmaswatch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "bulmaswatch*.+(js|css|map)"
```

Comment:
I can't use this rule to fetch files from GitHub.
The file structure is `[theme_name]/bulmaswatch.min.css` and
 `[theme_name]/bulmaswatch.min.css.map`

I'm not sure what rules should we apply here.
@pvnr0082t Please give some advices on this PR. Should we list all `theme_name`s ?